### ISSUE TITLE
[Question] Is there a max number of cards?

### ISSUE THREAD

**Issue by asddsaz** [megoforce/darkwinds#86]
**Question**: Is there a max number of cards? 
Like the max number that can be generated?

**Comment by rmujica** [comment]
yes, there is. for this edition is 1,000,000 cards. it is defined in the smart contract: https://github.com/megoforce/darkwinds/blob/master/contracts/DarkWindsFirstEdition.sol#L35
